android/app/build.gradle

```diff
@@ -91,6 +91,7 @@ dependencies {
     compile 'com.jakewharton:butterknife:8.4.0'
     compile 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
     apt 'com.jakewharton:butterknife-compiler:8.4.0'
+    compile 'net.opacapp:multiline-collapsingtoolbar:1.3.1'
```

Comment:
This is how it would look like if use custom Collapsing toolbar! we don't need to hide the menu items in this case.
It looks good to me. 
![screenshot from 2017-02-20 22 02 50](https://cloud.githubusercontent.com/assets/12701036/23133821/7fb3aa2a-f7b8-11e6-9d68-afa70b7e7a7b.png)
![screenshot from 2017-02-20 22 02 59](https://cloud.githubusercontent.com/assets/12701036/23133822/7fbda5de-f7b8-11e6-8b61-5dccb6673592.png)

What's your thought on this?